tests/test_robot_uds_auxiliary.py

```diff
@@ -170,3 +173,13 @@ def test_check_response_negative(robot_uds_aux, uds_aux):
     )
 
     assert ret is True
+
+
+def test_tester_present_sender(mocker, robot_uds_aux, uds_aux):
+    send_mock = mocker.patch.object(uds_aux, "send_uds_raw")
+    mocker.patch("time.sleep", return_value=None)
+
+    robot_uds_aux.start_tester_present_sender(1, "uds_aux")
```

Comment:
```suggestion
    robot_uds_aux.start_tester_present_sender(period=1, "uds_aux")
```

Reply:
Done

Reply:
Hum... actually does it really add anything to the unittest? Everything is mocked anyway